modules/manage/pages/cluster-maintenance/manage-throughput.adoc

```diff
@@ -2,17 +2,17 @@
 :description: Manage the throughput of Kafka traffic with configurable properties.
 :page-categories: Management, Networking
 
-Manage the throughput of Kafka traffic at the cluster level, with configurable properties that limit and protect the use of disk and network resources for individual brokers and for an entire cluster. Set broker-wide throughput limits for Kafka API traffic.
+Redpanda allows cluster administrators to configure throughput limits at both the broker level and the client level. These configurable limits help protect the use of disk and network resources. Limits applied on the broker apply to all clients that produce or consume there. Alternatively, the client limits allow you to selectively throttle a single client or group of clients.
 
 == Broker-wide throughput limits
 
 The network bandwidth and disk utilization of brokers may be overloaded by clients that produce or consume throughput without limits. To prevent resource overloading caused by unconstrained throughput and to configure back pressure, Redpanda provides runtime-configurable properties that limit and balance throughput of Kafka API traffic.
 
 To manage the volume of traffic going through a broker, Redpanda implements throughput quotas on the ingress and egress sides of every broker. The throughput quota accounts for all Kafka API traffic going in or out of a broker, with the value of quota representing the allowed rate of data passing through in one direction. When a connection is in breach of the quota, the throttler advises the client about the delay (throttle time) that would bring the rate back to the allowed level, and it implements that delay before handling Kafka API requests. To control the quotas, Redpanda provides configurable rate limits for total ingress and egress traffic through a broker.
 
-With Redpanda's xref:get-started:architecture.adoc#thread-per-core-model[thread-per-core model], the Kafka API traffic to and from a client connection is processed by a single core (shard). In order to manage throughput quotas efficiently, broker quotas are distributed between shards, and each per-shard quota is in turn shared by all connections served by the shard. Splitting broker quota optimally between shards is done behind the scenes by the quota balancer component.
+The throttling mechanism uses shared token buckets, one for ingress and one for egress. Tokens are added to the buckets at a constant rate and the number of tokens added is based on the amount of time elapsed since the last request for tokens. If the number of tokens in the bucket is below the `kafka_throughput_replenish_threshold`, then the bucket is topped up.
```

Comment:
```suggestion
The throttling mechanism uses shared token buckets, one for ingress and one for egress. Tokens are added to the buckets at a constant rate and the number of tokens added is based on the amount of time elapsed since the last request for tokens. If the number of tokens to be replenished is above `kafka_throughput_replenish_threshold` then the tokens are replenished.
```

Reply:
I've reworked this section so this language is no longer present.